apps/flutter_exercise/lib/views/widgets/login_widgets/checkbox.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+/// Flutter code sample for [Checkbox].
+
+
+class CheckboxExample extends StatefulWidget {
```

Comment:
@Mazit0 If you take a sample from internet, at least try to understand and refactor with your own programming style

Reply:
Entender lo entiendo a mi nivel, pq lo de los estados no me he metido a bichear pq estaba esperando tu explicación cuando tocase y aunque no refactorizase los nombres y eso, las propiedades del checkbox si que estan modificadas coño! xD